src/main/java/com/webperside/courseerpbackend/services/security/AuthBusinessService.java

```diff
@@ -22,6 +23,6 @@ public interface AuthBusinessService {
 
     void signUpOTPConfirmation(SignUpOTPRequest payload);
 
-    void setAuthentication(String email);
+    UserDetails setAuthentication(String email);
```

Comment:
is that fine for the function with the "setAuthentication" name to return UserDetails and behave like getUserDetails?